packages/expansion-hub/src/RevHub.ts

```diff
@@ -18,8 +18,8 @@ export interface RevHub {
 }
 
 export interface ParentRevHub extends RevHub {
-    getChildren(): ReadonlyArray<RevHub>;
+    readonly children: ReadonlyArray<RevHub>;
+    addChild(hub: RevHub): void;
     addChildByAddress(moduleAddress: number): Promise<RevHub>;
-
     readonly serialNumber: string
```

Comment:
Let's either put the properties together and the methods together, or organize everything alphabetically in one group. And be consistent with that between all of our interfaces.